### DISCUSSION TITLE
Filter Pull Requests shown on magit status screen to only mine

### DISCUSSION THREAD

**Discussion by gondalez** [magit/forge#668] [Ideas]
Hi :) 
Thanks very much for this excellent tool. I use and am delighted by it daily 🙏 

I notice if I type `@ t` on the magit screen I can easily see a list of my own pull requests for the repo.
The UI shows type: topics, subtype:pull requests, filter:created by default which is nice.

The feature I have in mind is to be able to filter the list of pull requests shown on the magit status screen to be only mine.

There is a similar command to filter out closed PRs by pressing `@ s c`. It is labeled "display recently closed topics".
Could such a filter live next to that ?
Maybe `@ s m` "display topics authored by me" (m mnemonic = "mine").

**Comment by tarsius** [comment]
That's part of a bigger change, I hope to land next week.

**Comment by tarsius** [comment]
It's now possible to filter the topics listed in the current buffer using `forge-topics-menu`. That menu is always available though the dispatch menu (`N` [^1] and then `m f`) and in topic list buffers also with a direct binding (`C-c C-c`).

[^1]: Or in your case `@` it seems.

**Reply by gondalez** [reply]
I got latest and can confirm it works nicely.
The nav is beautiful as always and I really appreciate the other new functionality.
Thanks 🙏

**Reply by tarsius** [reply]
Thanks! I hope many users find it intuitive enough, until I get around to the documentation. :grin:
